components/external-services-mock/cmd/main.go

```diff
@@ -410,6 +410,8 @@ func initDefaultCertServer(cfg config, key *rsa.PrivateKey, staticMappingClaims
 	router.HandleFunc("/formation-callback/async-no-config/{tenantId}", notificationHandler.AsyncNoConfig).Methods(http.MethodPatch)
 	router.HandleFunc("/formation-callback/async-no-response/{tenantId}", notificationHandler.AsyncNoResponseAssign).Methods(http.MethodPatch)
 	router.HandleFunc("/formation-callback/async-no-response/{tenantId}/{applicationId}", notificationHandler.AsyncNoResponseUnassign).Methods(http.MethodDelete)
+	router.HandleFunc("/formation-callback/async-new-ready/{tenantId}", notificationHandler.AsyncNoConfigWithCreateReady).Methods(http.MethodPatch)
+	router.HandleFunc("/formation-callback/async-new-ready/{tenantId}/{applicationId}", notificationHandler.AsyncNoConfigWithDeleteReady).Methods(http.MethodDelete)
```

Comment:
What do you think about `../async-create-ready/..` and `../async-delete-ready/..` for the paths?